content/kubeone/master/guides/addons/_index.en.md

```diff
@@ -62,7 +66,7 @@ the `addons` config:
 apiVersion: kubeone.io/v1beta1
 kind: KubeOneCluster
 versions:
-  kubernetes: 1.16.1
+  kubernetes: 1.20.1
```

Comment:
We can already put `1.22.1` here. :slightly_smiling_face:

Reply:
but 1.20.1 is still supported ;)